recipes/enum-flags/0.1a/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, tools
+import os
+
+
+class EnumFlagsConan(ConanFile):
+    name = "enum-flags"
+    description = "Bit flags for C++11 scoped enums"
+    homepage = "https://github.com/grisumbras/enum-flags"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("bitmask", "enum")
+    license = "MIT"
+    options = {"disable_implicit_conversion": [True, False]}
+    default_options = "disable_implicit_conversion=True"
```

Comment:
I always have trouble with the double negations. Can we change it to `enable_*` or is it a documented name for this option?

Reply:
Well, the macro name contains `FORBID`, so I figured it would be consistent with the library's choice.

Reply:
Also, `default_options` should be a dict.

Reply:
I would use the same then `forbid_implicit_conversion`